### ISSUE TITLE
tensorflow/colocate_with is deprecated and will be removed in a future version

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#2279]
The following deprecation need to be addressed before the temporary support is dropped:
~~~
sct_deepseg_gm -i t2s.nii.gz 
From /Users/julien/code/sct/python/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From /Users/julien/code/sct/python/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
~~~

**Comment by charleygros** [comment]
@jcohenadad: I could not find occurrences of `rate` (when calling `Dropout`) or `colocate_with` --> it's tensorflow code itself that has some occurrences of them.
I don't think we need to worry about this warning.

**Comment by jcohenadad** [comment]
right, it seems like this is going to be solved by the 1.14 or 2.0 version of TF. More info:
- https://github.com/tensorflow/tensorflow/issues/25996
- https://github.com/tensorflow/tensorflow/issues/25578

TF even shows the deprecated message in [their tutorial](https://www.tensorflow.org/tutorials/keras/save_and_restore_models) ;-)
